KarhooUISDK/CommonUI/VehicleCapacityView/NewVehicleCapacityView.swift

```diff
@@ -0,0 +1,133 @@
+            top: UIConstants.Spacing.xxSmall,
+            leading: UIConstants.Spacing.xSmall,
+            bottom: UIConstants.Spacing.xxSmall,
+            trailing: UIConstants.Spacing.xSmall
+        )
+        stack.spacing = UIConstants.Spacing.xxSmall
+    }
+
+    private lazy var baggageStack = UIStackView().then {stack in
+        stack.accessibilityIdentifier = KHNewVehicleCapacityViewID.baggageStack
+        stack.backgroundColor = KarhooUI.colors.background1
+        stack.clipsToBounds = true
+        stack.layer.cornerRadius = UIConstants.CornerRadius.xSmall
+        stack.isLayoutMarginsRelativeArrangement = true
+        stack.directionalLayoutMargins = NSDirectionalEdgeInsets(
+            top: UIConstants.Spacing.xxSmall,
+            leading: UIConstants.Spacing.xSmall,
+            bottom: UIConstants.Spacing.xxSmall,
+            trailing: UIConstants.Spacing.xSmall
+        )
+        stack.spacing = UIConstants.Spacing.xxSmall
+    }
+
+    private var baggageImageView: UIImageView = {
+        let imageView = UIImageView(image: UIImage.uisdkImage("luggage_icon"))
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFill
+        imageView.accessibilityIdentifier = KHNewVehicleCapacityViewID.baggageIcon
+        imageView.anchor(width: UIConstants.Dimension.Icon.medium, height: UIConstants.Dimension.Icon.medium)
+        return imageView
+    }()
+
+    private lazy var baggageCapacityLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.accessibilityIdentifier = KHNewVehicleCapacityViewID.baggageCapacityLabel
+        label.textColor = KarhooUI.colors.primary
+        label.font = KarhooUI.fonts.footnoteRegular()
+        return label
+    }()
+
+    private lazy var passengerCapacityImageView: UIImageView = {
+        let imageView = UIImageView(image: UIImage.uisdkImage("passenger_capacity_icon"))
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFill
+        imageView.accessibilityIdentifier = KHNewVehicleCapacityViewID.capacityIcon
+        imageView.anchor(width: UIConstants.Dimension.Icon.medium, height: UIConstants.Dimension.Icon.medium)
+        return imageView
+    }()
+
+    private lazy var passengerCapacityLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.accessibilityIdentifier = KHNewVehicleCapacityViewID.passengerCapacityLabel
+        label.textColor = KarhooUI.colors.primary
+        label.font = KarhooUI.fonts.footnoteRegular()
+        return label
+    }()
+
+    // MARK: - Init
+    init() {
+        super.init(frame: .zero)
+        self.setUpView()
+    }
+
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+    private func setUpView() {
```

Comment:
Please split this method to keep SRP followed

Reply:
done